target/postfix/main.cf

```diff
@@ -57,6 +57,12 @@ smtpd_sender_restrictions = $dms_smtpd_sender_restrictions
 smtpd_discard_ehlo_keywords = silent-discard, dsn
 disable_vrfy_command = yes
 
+# Security - Prevent SMTP Smuggling attack
+# https://www.postfix.org/smtp-smuggling.html#long
+smtpd_forbid_bare_newline = yes
+# It is possible to exclude clients on trusted networks from this restriction:
+# smtpd_forbid_bare_newline_exclusions = $mynetworks
```

Comment:
The default is actually `$mynetworks` already, perhaps we want to disable that? Or rely on the default of `PERMIT_DOCKER=none` making that unnecessary?

Only concern is due to Docker and IPv6 capable hosts mistakenly routing through to an IPv4 only DMS container (_due to Docker daemon `user-proxy: true` default config_), where it'll be trusted in `$mynetworks` that cover the Docker network gateway address in a subnet (eg: `x.y.z.1`).